ansible/roles/test/tasks/acltb.yml

```diff
@@ -76,18 +76,54 @@
         errors_expected: false
       include: roles/test/tasks/run_command_with_log_analyzer.yml
 
+    - name: Get the total ACL table size before populating the ACL table
+      command: "{{item}}"
+      with_items:
+        - crm config polling interval 5
+        - crm show resources acl table
```

Comment:
Suggest to pause 5 seconds between the two commands. i.e fetching the resource and setting the polling interval. Also ensure that the polling interval is reset back once the test is finished